### PR TITLE
Recursive and iterative working

### PR DESCRIPTION
**Pull request by kozlofski**

### CODE REVIEW

File: module1/homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if(sequence == 0) {
+        return 0;
+    }
+    if(sequence < 3) {
+        return 1;
+    } else {
+        int minus_two = 1;
```
Comment (kozlofski) [inline module1/homework/fibonacci/fibonacci.hpp:10]: sam sobie skomentowałem

File: module1/homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,26 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if(sequence == 0) {
+        return 0;
+    }
+    if(sequence < 3) {
+        return 1;
```
Comment (nauka-programowania-MA) [inline module1/homework/fibonacci/fibonacci.hpp:8]: fibb for 2 is 2

Comment (kozlofski) [reply module1/homework/fibonacci/fibonacci.hpp:8]: Sorry, but you're not right. The sequence is: 1, 1, 2, 3, 5, 8, etc. (or 0, 1, 1, 2, 3 ...)

Comment (nauka-programowania-MA) [reply module1/homework/fibonacci/fibonacci.hpp:8]: Yes You are right :) My bad, good work!

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1430], "points": [2, 6] }

**Pull request closed without merging**